.github/issue_template.md

```diff
@@ -1,14 +1,15 @@
 ### Steps to reproduce
-How can we make this happen
+<!-- How can we make this happen -->
```

Comment:
Not sure where/what changes having this text as comment, can you please explain better?

Reply:
Sorry, here is the explanation in the commit message:

>Add comments to the issue template
>
>Personally I always end up removing these instructions so it made
sense to me to go ahead and make them comments. This commit also
adds instructions to get the current Solidus version from Rails
console.

This just makes them not show up in the issue if someone doesn't remove them. It's purely my preference and I'm happy to remove that change.

Reply:
I did read the commit description but still can't understand the final effect of this. Will that text be invisible in GH? If yes, what about the `Steps to reproduce` title that will still be visible without any content inside?

Reply:
### Steps to reproduce
<!-- How can we make this happen -->
Example

### Expected behavior
<!-- Tell us what should happen -->
Example

### Actual behavior
<!-- Tell us what happens instead -->
Example

### System configuration
**Solidus Version**:
<!-- Unsure? Check with `Spree.solidus_version` -->
2.7.0
**Extensions in use**:
N/A

# Source for this comment:
```
### Steps to reproduce
<!-- How can we make this happen -->
Example

### Expected behavior
<!-- Tell us what should happen -->
Example

### Actual behavior
<!-- Tell us what happens instead -->
Example

### System configuration
**Solidus Version**:
<!-- Unsure? Check with `Spree.solidus_version` -->
2.7.0
**Extensions in use**:
N/A
```

Reply:
This is how it should work.

Reply:
@kennyadsl Like I said, I'm am happy to get rid of this change if you think it doesn't work.

Reply:
Gotcha, thanks for the explanation. I have no strong opinion around that but I'm seeing that it's also used [here](https://github.com/stevemao/github-issue-templates) so 👍 . What about opening a new PR for this? I think it's not really related to this PR and could make sense to have a separate discussion around that.

Reply:
@kennyadsl done!